ui/src/app/pages/data/concept/concept-homepage.spec.tsx

```diff
@@ -30,6 +30,7 @@ import {
   WorkspaceStubVariables
 } from 'testing/stubs/workspaces-api-stub';
 import {Key} from 'ts-key-enum';
+import {CohortPage} from '../../../cohort-search/cohort-page/cohort-page.component';
```

Comment:
nit:path

Reply:
That shouldn't even be there, removed. Surprised the test didn't fail because of this 🤷‍♂️